### DISCUSSION TITLE
Import an existing database

### DISCUSSION THREAD

**Discussion by Sunsoft-io** [planetscale/discussion#132] [FAQs] [answered]
Hello,

I would like to import an existing database via the webinterface (as in phpMyAdmin). I've searched the documentation, but there was no entry.

Thanks...Willi Rauffer, `link-removed`

**Comment by jgreet** [comment, marked as answer]
Hi, we launched an import feature for existing MySQL databases in the PlanetScale web app today:

https://planetscale.com/blog/import-your-mysql-data-to-planetscale
https://docs.planetscale.com/concepts/database-imports

Let us know if that works for you.

**Reply by jonrcrowell** [reply]
I'm trying to import a database that is hosted on digital ocean. I've entered all the requested information in the Import External Database form and get a Successfully Connected message with a green check mark. Unfortunately, when I click the Begin Database Import button, the connection fails and the Successfully Connected message is replaced with a Could Not Connect to the Database message.
![data import error](https://user-images.githubusercontent.com/14331191/142473083-79caff4a-6a5d-497a-9dad-87f2f1be7e95.png)

**Reply by jgreet** [reply]
@jonrcrowell which organization on PlanetScale are you attempting the import in?

**Reply by jonrcrowell** [reply]
stone-giant-studio
